content/client-server-api/modules/spaces.md

```diff
@@ -2,8 +2,8 @@
 
 {{% added-in v="1.2" %}}
 
-Often used to group rooms of similar subject matter (such as a public "Official
-matrix.org rooms" space or personal "Work stuff" space), spaces are a way to
+Often used to group rooms of similar subject matter (such as an "Official
+matrix.org rooms" space or a "Work stuff" space), spaces are a way to
```

Comment:
The public or personal nature seemed irrelevant in these examples. It's also implied by the space names already.

Reply:
Work stuff sounds like either public or personal to me

Reply:
Ok, yes. But do you think the access level is relevant in this paragraph?